pymc3/tests/test_transforms.py

```diff
@@ -93,6 +93,17 @@ def check_jacobian_det(transform, domain,
             actual_ljd(yval),
             computed_ljd(yval), tol)
 
+def test_ordered():
+    check_vector_transform_identity(tr.ordered, Vector(R, 6))
```

Comment:
The domain of ordered transform is not real value, but sorted real value. You should create a new domain `SortedVector` here.

Reply:
Maybe something like:
```python
# maybe this could be put into test_distributions.py
def SortedVector(n):
    vals = []
    np.random.seed(42)
    for _ in range(10):
        vals.append(np.sort(np.random.randn(n)))
    return Domain(vals, edges=(None, None))

def test_ordered():
    check_vector_transform_identity(tr.ordered, SortedVector(6))
```